czarnowiejska.py

```diff
@@ -138,7 +138,7 @@
     *NAWOJKI_SECOND_TURN_RIGHT,
     *NAWOJKI_SECOND_TURN_LEFT,
 
-    *NAWOJKI_FIRST_TURN_RIGHT,
+    # *NAWOJKI_FIRST_TURN_RIGHT,

```

Comment:
It should be commented?

Reply:
Yeah, we consider to delete it anyway later